explorer/src/components/Domain/query.ts

```diff
@@ -74,10 +74,10 @@ export const QUERY_DOMAIN_STATUS = gql`
   query DomainsStatus(
     $limit: Int!
     $offset: Int
-    $orderBy: [domain_order_by!]!
-    $where: domain_bool_exp
+    $orderBy: [staking_domains_order_by!]!
+    $where: staking_domains_bool_exp
   ) {
-    domain(order_by: $orderBy, limit: $limit, offset: $offset, where: $where) {
+    staking_domains(order_by: $orderBy, limit: $limit, offset: $offset, where: $where) {
```

Comment:
why this change on the field name? It seemed for intuitive `domain`

Reply:
We used to have 4 different endpoints for the 4 different micro squids, and each time we will use apollo, we had to tell it what context (what api endpoint to use) this was causing some issues as if a request was not yet resolved, and you changed page, to a new context, it will cause errors. and it was not allowing us to do the relation between these different dbs.

The new setup has multiple indexer all indexing on the same postgres in different tables, and hasura allowing us to create relation between these tables, so we have consensus, leaderboard and staking and to not have everything mix up, all the tables from a given indexers are prefix with the indexers they are from